hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/RSRpcServices.java

```diff
@@ -1291,7 +1295,8 @@ protected RpcServerInterface createRpcServer(Server server, Configuration conf,
   }
 
   protected Class<?> getRpcSchedulerFactoryClass() {
-    return this.regionServer.conf.getClass(REGION_SERVER_RPC_SCHEDULER_FACTORY_CLASS,
```

Comment:
it is also ok if not change?

Reply:
True. Breaking up these compound statements makes it easier to see NPEs when they happen. I guess such stylistic choices will become obsolete once Java14 lands.

https://bugs.openjdk.java.net/browse/JDK-8218628

Reply:
Great.